### PR TITLE
{bio}[intel/2020b] loompy v3.0.6 w/ Python 3.8.6

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
@boegelbot please test @ generoso

File: easybuild/easyconfigs/l/loompy/loompy-3.0.6-intel-2020b.eb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+        'checksums': ['7b17b291322353f07c51598512d077e3731da0a048cfa8f738f3460d1ef0658d'],
+    }),
+    (name, version, {
+        'checksums': ['58e9763b8ab1af2a4a0e3805d120458b5184fd2b0f3031657ecce33c63ca4c46'],
+    }),
+]
+
+sanity_check_paths = {
+    'files': ['bin/loompy'],
+    'dirs': ['lib/python3.8/site-packages'],
```
Comment (Micket) [inline easybuild/easyconfigs/l/loompy/loompy-3.0.6-intel-2020b.eb:31]:
```suggestion
    'dirs': ['lib/python%(pyshortver)s/site-packages'],
```

**Review by Micket — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Review by Micket — APPROVED** [review]
lgtm

File: easybuild/easyconfigs/l/loompy/loompy-3.0.6-intel-2020b.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/loompy/loompy-3.0.6-intel-2020b.eb]: Ouch, thanks for fixing this @Micket !

**Pull request merged**